### ISSUE TITLE
Error loading the dynamic library in PHP binding

### ISSUE THREAD

**Issue by peter279k** [meganz/sdk#515]
Hi everyone. 
After I compile and install the MEGA SDK with enabling PHP successfully, I'm failed to run the sample code: ```megacli.php```.
I got the warning message about this: ```PHP Warning:  dl(): Unable to load dynamic library '/usr/lib/php/20131226/libmegaphp.so' - libmega.so.20800: cannot open shared object file: No such file or directory in /home/lab223/sdk-2.8.0/bindings/php/megaapi.php on line 4```

What should I do?
BTW, my PHP version is PHP 5.6 and use the latest version: 2.8.0.

Thanks.

**Comment by javiserrano** [comment]
It's needed to enable the load of shared libraries in your `php.ini`changing the `enable_dl` option from `Off` to `On`. If it fails after that, let us know and we will test with that PHP version.

**Comment by peter279k** [comment]
Hi @javiserrano, thank you for your comment. I still get the same error message after change the ```enable_dl``` setting value.

I use the PHP version is 5.6.29 and I build this SDK on my Linux/Ubuntu.

Should I need to add the ```extension``` setting value in ```php.ini``` ?Thanks.

**Comment by javiserrano** [comment]
I'm not sure. 
Which version of Ubuntu are you using? Are you using the default PHP package of that Ubuntu version? That information will help us run our tests exactly in the same environment.

**Comment by peter279k** [comment]
Hi @javiserrano, I use the Ubuntu 16.04.01 LTS. Here is my build script. Thanks.

https://gist.github.com/peter279k/8e2dcff57fe01ad63e75a7b59a911168

**Comment by peter279k** [comment]
I update my new progress now. I set up a new Ubuntu and rebuild the MEGA SDK.
It still got the same error message. I also check the message if I change the ```enable_dl=On``` to ```enable_dl=Off```. The error message will be:

```
PHP Warning:  dl(): Dynamically loaded extensions aren't enabled in /home/lee/sdk-2.8.0/bindings/php/megaapi.php on line 4
```
I think some configurations has error or bugs. Thanks.

**Comment by peter279k** [comment]
Hi @javiserrano, thank you for your comment.
Finally I found the error and I have fixed it now.
The problem is: the shared objects is installed in the ```/usr/lib/``` or ```/lib``` directory.
So we need to execute the command: ```sudo ldconfig``` and allow to search the ```/lib``` and ```/usr/lib``` directories. It also load the ld configuration files.

Now my completed build scripts is available [here](https://gist.github.com/peter279k/8e2dcff57fe01ad63e75a7b59a911168).

Maybe I will improve and add the build section in README.md.

Thank you for your help again.
